.github/workflows/api_pipeline.yml

```diff
@@ -22,7 +26,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install black
+        pip install black==19.10b0
```

Comment:
If we do `pip install -r requirements-dev.txt` it will install the same version of black as we have on local, so we don't have to keep the version in sync. It would mean that this job would take longer though.

Reply:
A quicker (and probably more dangerous) idea would be to do something like this to get the version of Black we are using:
```sh
cat ./api/requirements-dev.txt | grep black== | sed -e 's/black==\(.*\)/\1/' | cut -d' ' -f1
```
Now, I have a good feeling this is a bad idea, I just want to be sure it is 🤔

Reply:
That is not a bad idea, but I don't think you need `sed`.  This should do it:
```bash
cat api/requirements-dev.txt | grep black== | cut -d' ' -f1 | xargs pip install
```